src/test/java/teammates/test/cases/browsertests/InstructorCourseEnrollPageUiTest.java

```diff
@@ -69,16 +69,20 @@ private void testEnrollAction() throws Exception {
         enrollPage = loginAdminToPage(enrollUrl, InstructorCourseEnrollPage.class);
 
         String enrollString =
-                "Section | Team | Name | Email | Comments\n"
+                "Section|Team|Name|Email|Comments\n"
```

Comment:
Any reason for these changes? those random spaces could have been intentional.

Reply:
Since the user is not interacting with the textarea we don't need these spaces, they were for better visibilty of the text.

Reply:
You may not want to remove all of them though, because some of them ensures that the code still works even if there are extra spaces around the values.